apps/crisis-unleashed/components/admin/favicon/renderers/standard-favicon-renderer.ts

```
@@ -0,0 +1,47 @@
+      case "eclipsed-order":
+        renderEclipsedOrder(props);
+        break;
+      case "titanborn":
+        renderTitanborn(props);
+        break;
+      case "celestial-dominion":
+        renderCelestialDominion(props);
+        break;
+      default:
+        // Default fallback - draw a simple circle
+        const { size, innerSize } = props;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)